codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/swiftmodules/AWSClientRuntimeTypes.kt

```diff
@@ -62,6 +62,7 @@ object AWSClientRuntimeTypes {
             runtimeSymbol("FlexibleChecksumsResponseMiddleware", SwiftDeclaration.STRUCT)
         val AmzSdkInvocationIdMiddleware = runtimeSymbol("AmzSdkInvocationIdMiddleware", SwiftDeclaration.STRUCT)
         val AmzSdkRequestMiddleware = runtimeSymbol("AmzSdkRequestMiddleware", SwiftDeclaration.CLASS)
+        val AWSClient = runtimeSymbol("AWSClient", SwiftDeclaration.PROTOCOL)
```

Comment:
This is a symbol for a new protocol for AWS service clients that is defined below.